src/standalone.js

```diff
@@ -54,9 +54,17 @@ export default function ({
       })
     }
 
-    const chosenFormatter = (typeof formatter === "string")
-      ? formatters[formatter]
-      : formatter
+    let chosenFormatter = ""
```

Comment:
This is kind of a confusing default value. I'd say leave it undefined (`let chosenFormatter`) or set `let chosenFormatter = formatter` and just change it if it's a string.